src/cargo/core/compiler/fingerprint/mod.rs

```diff
@@ -1701,6 +1725,7 @@ fn build_script_local_fingerprints(
     // obvious.
     let pkg_root = unit.pkg.root().to_path_buf();
     let target_dir = target_root(build_runner);
+    let env_config = Arc::clone(build_runner.bcx.gctx.env_config().unwrap());
```

Comment:
Whats the assumption we are operating under that its safe to `unwrap`?

Reply:
Updated.